Lesson5/src/app/pages/Post.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import axios from 'axios';
+import PostContent from '../components/Post';
+import CommentList from '../components/CommentList';
+
+export default class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      post: null
+    };
+  }
+
+  render()
+  {
+    
+    return(
+      <div>
+        {this.state.post && <PostContent {...this.state.post}/>}
+        <hr/>
+        <CommentList post={this.state.post}/> 
```

Comment:
```suggestion
        {this.state.post && this.state.post.id ? <CommentList post={this.state.post.id}/> : null}
```

просто в начале `this.state.post` возвращает `undefined`, а `undefined.id` выдает ошибку, поэтому и не получалось:)